src/main/api/2/studio-api-2.yaml

```diff
@@ -6330,6 +6330,197 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /workflow/request_publish:
+    post:
+      tags:
+        - workflow
+      summary: Request publish for content
+      description: 'Required permission "site_member"'
+      operationId: workflowRequestPublish
+      requestBody:
+        description: parameters for request publish
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties: 
+                items:
+                  type: array
+                  description: path(s) of content item(s)
+                  items:
+                    type: string
+                optional:
+                  type: array
+                  description: optional (soft) dependencies
+                  items:
+                    type: string
+                mandatory:
```

Comment:
Remove `mandatory and instead calculate this on the backend and call it internally `mandatoryDependencies`.